.storybook/main.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
         },
       },
     },
+    'storybook-addon-next-router',
```

Comment:
storybook needed some config to work with next/router since the router requires context vars